pkg/kudoctl/cmd/init/crds.go

```diff
@@ -93,7 +93,7 @@ func generateOperator() *apiextv1beta1.CustomResourceDefinition {
 		"description":       apiextv1beta1.JSONSchemaProps{Type: "string"},
 		"kubernetesVersion": apiextv1beta1.JSONSchemaProps{Type: "string"},
 		"kudoVersion":       apiextv1beta1.JSONSchemaProps{Type: "string"},
-		"maintainers": apiextv1beta1.JSONSchemaProps{Type: "object",
+		"maintainers": apiextv1beta1.JSONSchemaProps{Type: "array",
```

Comment:
Nice. Would you mind adding maintainers to the [test-operator.yaml](https://github.com/kudobuilder/kudo/blob/master/test/manifests/test-operator.yaml) so we have this covered in a test?

Reply:
sure thing!